canonical-websites/www.ubuntu.com
# Update /download/server/provisioning to vbt

## Done

* update /download/server/provisioning to vbt
* moved marketo js to extra footer block

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/download/server/provisioning)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/9.%20Download?preview=download-server-provisioning.png)

## Issue / Card

Partially fixes #2004
